server/superdesk/macros.py

```diff
@@ -21,6 +22,13 @@ def load_macros():
     """
     try:
         module = app.config['MACROS_MODULE']
+        already_imported = sorted(sys.modules.keys())
+        for t in already_imported:
+            if t == module:
+                m = sys.modules[t]
+                imp.reload(m)
```

Comment:
`importlib.reload` is only available from 3.4 forward. So it would fail in 3.3 which I use for development.